modules/primer-css/CHANGELOG.md

```diff
@@ -0,0 +1,305 @@
+# 9.6.1
```

Comment:
Don't put this file here, the build will publish the changelog. I think I might have done something wrong with the 9.6 release.